pkgs/development/compilers/zig/bintools.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  zig,
+  runCommand,
+  makeWrapper,
+}:
+let
+  targetPrefix = lib.optionalString (
+    stdenv.hostPlatform != stdenv.targetPlatform
+  ) "${stdenv.targetPlatform.config}-";
+in
+runCommand "zig-bintools-${zig.version}"
+  {
+    pname = "zig-bintools";
+    inherit (zig) version meta;
+
+    nativeBuildInputs = [ makeWrapper ];
+
+    passthru = {
+      isZig = true;
+      inherit targetPrefix;
+    };
+
+    inherit zig;
+  }
+  ''
+    mkdir -p $out/bin
+    for tool in ar objcopy; do
+      makeWrapper "$zig/bin/zig" "$out/bin/${targetPrefix}-$tool" \
```

Comment:
Shouldn't it be `${targetPrefix}$tool` (without the -)?

Reply:
Yes, that was a typo.